core/miner/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ add_subdirectory(storage_fsm)
 
 add_library(miner
         impl/miner_impl.cpp
-    )
+        )
```

Comment:
There were problems with indents in this file, please, use Ctrl+Alt+L for the whole file.

Reply:
Just revert/remove this unnecessary change